README.md

```diff
@@ -72,7 +72,15 @@ black .
 
 ### `/listeners`
 
-Every incoming request is routed to a "listener". Inside this directory, we group each listener based on the Slack Platform feature used, so `/listeners/events` handles incoming events, `/listeners/shortcuts` would handle incoming [Shortcuts](https://api.slack.com/interactivity/shortcuts) requests, and so on.
+Every incoming request is routed to a "listener". Inside this directory, we group each listener based on the Slack Platform feature used, so `/listeners/events` handles incoming events like messages sent to the app.
+
+**Note**: The `listeners/events` folder is purely educational and demonstrates alternative implementation approaches. These listeners are **not registered** and are not used in the actual application. For the working implementation, refer to `listeners/assistant.py`.
+
+**`/listeners/assistant`**
+
+Configures the new Slack Assistant features, providing a dedicated side panel UI for users to interact with the AI chatbot. This includes:
+*  `@assistant.thread_started` - Manages when users start new assistant threads. 
+*  `@assistant.user_message` - Processes user messages in assistant threads and app DMs. **Replaces traditional DM handling as seen in** `/listeners/events/user_message.py`
```

Comment:
```suggestion
*  `@assistant.user_message` - Processes user messages in app threads or from the app **Chat** and **History** tab.
```

🪓 suggestion: We should match the wording on upstream docs for consistent verbiage and not to confuse messages to the app otherwise

🔗 https://docs.slack.dev/ai/developing-ai-apps#agents--ai-apps-feature